composer.json

```diff
@@ -14,26 +14,29 @@
     }
   },
   "require": {
-    "php": "~7.2",
+    "php": "^8.1",
     "ext-openssl": "*",
     "beberlei/assert": "^3",
-    "sensio/framework-extra-bundle": "^5.4",
-    "symfony/monolog-bundle": "^3.6.0",
-    "surfnet/stepup-saml-bundle": "^5.0",
-    "symfony/dependency-injection": "^4.4",
-    "symfony/framework-bundle": "^4.4"
+    "symfony/monolog-bundle": "^3.8",
+    "surfnet/stepup-saml-bundle": "^6.0",
+    "symfony/dependency-injection": "^5.4|^6.3",
+    "symfony/framework-bundle": "^5.4|^6.3"
   },
   "require-dev": {
-    "behat/behat": "^3.5",
+    "behat/behat": "^3.13",
     "jakub-onderka/php-parallel-lint": "^1",
     "malukenho/docheader": "^0",
-    "mockery/mockery": "^1",
-    "phpmd/phpmd": "^2",
-    "phpunit/phpcov": "^6",
-    "phpunit/phpunit": "^8",
-    "sebastian/phpcpd": "^4",
-    "squizlabs/php_codesniffer": "^3",
-    "symfony/phpunit-bridge": "~4"
+    "mockery/mockery": "^1.5",
+    "overtrue/phplint": "*",
```

Comment:
No exact version? Consider changing this to ^9.0

Reply:
I think this was wildcarded to allow a suitable version for both SF 5 and 6 users.

Reply:
But not anymore? Keeping the wildcard?